sopel/bot.py

```diff
@@ -376,6 +376,11 @@ def reply(self, text, dest, reply_to, notice=False):
         else:
             self.say(text, dest)
 
+    def kick(self, text, channel, nick):
```

Comment:
`text` should be the last argument, and default to `''` or `None`, so it can be omitted. The `KICK` IRC command itself only requires `channel` and `nick`.